_FIPS201/requirements.md

```diff
@@ -26,8 +26,7 @@ accreditation process.
 Each agency's PIV implementation **SHALL** meet the four control objectives (a) through (d) listed above
 such that
 
-+ A credential is issued only to an individual whose identity has been verified and who has been appropriately vetted as per [Section 2.2](requirements.md#s-2-2)
-    after a proper authority has authorized issuance of the credential.
++ A credential is issued to an individual only after a proper authority has authorized issuance of the credential, the individual's identity has been verified, and the individual has been vetted per [Section 2.2](requirements.md#s-2-2).
```

Comment:
This section still seems like it needs a little work. Isn't the "vetted" part covered by the following bullet regarding adjudication of federal investigation? That following bullet is better because it contains the language allowing the issuance of an interim credential while the BI is in progress.

The "identity has been verified" part also overlaps with bullets 3 and 4, except that they don't cover verification that the individual presenting the documents match the documents themselves.

There is also no clear statement that all of the bullets need to be satisfied (probably obvious, but we shouldn't depend on that). So maybe the introductory text should say something like, "...such that all of the following SHALL be satisfied:"

Reply:
That would then read: "Each agency's PIV implementation SHALL meet the four control objectives (a) through (d) listed above such that all of the following SHALL be satisfied."  The two shalls seem a bit funny to me.  Isn't the shall already implied?

Regarding the overlap, it seems like 2-5 are sort of sub-items under the first item.  Would making them sub-bullets be crazy?